### DISCUSSION TITLE
How to get ServerCallId for recording

### DISCUSSION THREAD

**Discussion by Ferb-dee** [Azure/communication-ui-library#2384] [Q&A]
Hi Guys

How would I get The ServerCallId, I would like to record the calls and save them, using this sample.

**Comment by prabhjot-msft** [comment]
Hi @Ferb-dee, thanks for the question. Once the call is connected you can use the `getServerCallId` method to get the server call ID. For detailed information you can refer to this document https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/call-recording-sample?pivots=programming-language-java#getting-servercallid-as-a-requirement-for-call-recording-server-apis-from-javascript-application

**Comment by alkwa-msft** [comment]
Closing this discussion due to inactivity. Feel free to re-open if this is still a topic we would like to continue.

**Reply by VishalMauryastp** [reply]
How we can record a call in ACS
